config/environments/test.rb

```diff
@@ -58,4 +58,7 @@
 
   # Raises error for missing translations
   # config.action_view.raise_on_missing_translations = true
+
+  # set fake_data for pam, don't do real calls, just use fake data
+  Rpam2.fake_data = { usernames: Set['pam_user1', 'pam_user2'], servicenames: Set['pam_test', 'pam_test_controlled'], password: '123456', env: { email: 'me@example.com'} }  if AppConfig.pam.enable?
```

Comment:
- Space inside { detected.
- Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.
- Line is too long. [197/120]
- Space inside } detected.
- Unnecessary spacing detected.